examples/asr/speech_to_text.py

```diff
@@ -67,6 +69,14 @@ def main(cfg):
     trainer = pl.Trainer(**cfg.pl.trainer)
     exp_manager(trainer, cfg.get("exp_manager", None))
     asr_model = EncDecCTCModel(cfg=cfg.model, trainer=trainer)
+
+    if 'logger' in cfg.model:
+        if cfg.model.logger.experiment_name is not None and cfg.model.logger.project_name is not None:
+            logger = WandbLogger(name=cfg.model.logger.experiment_name, project=cfg.model.logger.project_name)
+            trainer.configure_logger(logger)
```

Comment:
Eh, this probably isn't going to work with exp_manager. Should we add WandbLogger to exp_manager?
Alternatively, can you check that `trainer.logger` is None before doing this line?

Reply:
I actually tried a run with WandB and it does work, but weights get stored into a weird name hash directory instead of expmanager directory. Maybe we should integrate WandB, or at the least configure wandb paths based on exp_manager

Reply:
Let me look into how to get path from `trainer.logger` and attach that to wandb.

Reply:
What I mean is if I don't want to use WandBLogger and instead want to use TensorboardLogger. Doing this this way will cause WandBLogger to overwrite the `trainer.logger` object that was supposed to point to TensorboardLogger.
Which is why I propose to check that `trainer.logger is None` before doing `trainer.configure_logger`.

Reply:
So I was able to merge this logic with exp_manager by adding 2 more arguments (optional and checked at runtime so no effect to older exp_manager config).

Btw, while looking into this, we have to be very careful when we add more loggers. The default LoggerCollection in PTL actually makes a mess of the directory structure so I had to subclass and override its @parameters to have safe behaviour. The subclass, `LoggerList` should be used, and 1st argument should always be the TensorboardLogger.

Reply:
Hmm interesting, but does lightning use logger.name and logger.version anywhere?

Reply:
PTL seems to have those two values for all inheritors of LoggerBase, and uses the name for constructing the directories.